modules/data.remote/R/LandTrendr.AGB.R

```diff
@@ -220,23 +305,25 @@ download.LandTrendr.AGB <- function(outdir, target_dataset = "biomass", product_
 ##' ids = site_ID, .con = con))
 ##' suppressWarnings(qry_results <- DBI::dbSendQuery(con,site_qry))
 ##' suppressWarnings(qry_results <- DBI::dbFetch(qry_results))
-##' site_info <- list(site_id=qry_results$id, site_name=qry_results$sitename, lat=qry_results$lat, 
-##' lon=qry_results$lon, time_zone=qry_results$time_zone)
 ##' data_dir <- "~/scratch/agb_data/"
 ##' 
 ##' results <- extract.LandTrendr.AGB(site_info, "median", buffer = NULL, fun = "mean", 
 ##' data_dir, product_dates, output_file)
 ##' 
 ##' }
 ##' 
+##' @return list of two containing the median AGB values per pixel and the corresponding 
+##' standard deviation values (uncertainties)
+##' 
 ##' @export
 ##' @author Shawn Serbin, Alexey Shiklomanov
 ##' 
-extract.LandTrendr.AGB <- function(site_info, dataset = "median", buffer = NULL, fun = "mean", 
+extract.LandTrendr.AGB <- function(site_info=NULL, dataset = "median", buffer = NULL, fun = "mean", 
                                    data_dir = NULL, product_dates = NULL, output_file = NULL, 
                                    ...) {
 
-  ## get coordinates and provide spatial info
+  ## get coordinates and provide spatial info - should harmonize what packages we use in all data.remote functions
+  site_info <- site_info
   site_coords <- data.frame(site_info$lon, site_info$lat)
```

Comment:
This line will crash if site_info is NULL. To me, it seems like you'd want to revert your change above that allows site_info to be NULL, but if you decide to keep it you need to add a lot of error checking for what to do if the site is null.